term.go

```diff
@@ -126,7 +126,12 @@ func (t *Terminal) MinSize() fyne.Size {
 // MouseDown handles the down action for desktop mouse events.
 func (t *Terminal) MouseDown(ev *desktop.MouseEvent) {
 	if t.hasSelectedText() {
+		t.copySelectedText(fyne.CurrentApp().Clipboard())
 		t.clearSelectedText()
+	} else {
+		if ev.Button == desktop.MouseButtonSecondary {
```

Comment:
Why not just make the "else" an "else if"?

Reply:
No real reason, other than I'm not a huge fan of `else if` as I feel the nesting gives better readability.

Reply:
That's very much the opposite opinion from what I have. I feel that the free levels of nesting, the better. Less horizontal space to parse but I guess there is not right or wrong.

Reply:
I appreciate that this could be subjective - but it is general Fyne style guide that we keep indentation to a minimum (through shorter methods and early exiting mostly).

Reply:
I agree 100% ill refactor this to else if, if I can't think of a better way/ nicer read